src/component/base/DraftEditor.react.js

```diff
@@ -190,6 +191,10 @@ class DraftEditor extends React.Component {
       if (!this.props.readOnly) {
         const method = this._handler && this._handler[eventName];
         method && method(this, e);
+      } else if (eventName === 'onCopy') {
+        editOnSelect(this)
```

Comment:
onSelect doesn't even get this far so I'm unable to let it go through. Something else prevents `focus` and `select` from firing, regardless of the readonly check here. I started to go down that rabbit hole but wasn't making progress, so went this route.